lib/jsdom/living/webstorage/Storage-impl.js

```diff
@@ -1,10 +1,12 @@
 "use strict";
 
 const DOMException = require("domexception/webidl2js-wrapper");
-const StorageEvent = require("../generated/StorageEvent");
-const idlUtils = require("../generated/utils");
+
 const { fireAnEvent } = require("../helpers/events");
 
+const idlUtils = require("../generated/utils");
+const StorageEvent = require("../generated/StorageEvent");
```

Comment:
Is&nbsp;it&nbsp;necessary to&nbsp;move&nbsp;this after&nbsp;`fireAnEvent` and&nbsp;`idlUtils`?

Reply:
That's an old change that I haven't reverted properly.